packages/ecommerce-app-base/src/Editor/SortableComponent.stories.tsx

```diff
@@ -1,15 +1,20 @@
+import React from 'react';
 import type { Meta, StoryObj } from '@storybook/react';
 import { productsList } from '../__mocks__';
-import { SortableList } from './SortableList';
 import { integration } from '../__mocks__/storybook/integration';
 import { IntegrationProvider } from './IntegrationContext';
-import { Integration } from '../types';
+import { Integration, Product } from '../types';
 import { decorators } from '../__mocks__/storybook/decorators';
+import { SortableComponent } from './SortableComponent';
+import { sdk } from '../__mocks__/storybook/sdk';
+import { cloneDeep } from 'lodash';
 
-type Args = typeof SortableList & Pick<Integration, 'productCardVersion'>;
+const productPreviews: Product[] = cloneDeep(productsList);
```

Comment:
This is tough, because of the way Storybook is designed to detect re-rendering changes.  Storybook will rerender when `args` are updated, but since product previews is defined outside of the direct arg attributes, changing `productPreviews` does not cause a re-render.

I spend about an hour on this and couldn't get it to work, but it feels like a pretty low yield effort so just going to add a console.log for now and move on.